lib/org/chromium/webidl/PipelineMessage.js

```diff
@@ -39,5 +39,20 @@ foam.CLASS({
         following types: Interface | Callback | Namespace | Dictionary | Enum |
         Typedef | ImplementsStatement | [ Enum | Typedef ].`,
     },
+    {
+      class: 'Enum',
+      of: 'org.chromium.webidl.WebPlatformEngine',
+      name: 'leftSource',
+    },
+    {
+      class: 'Enum',
+      of: 'org.chromium.webidl.WebPlatformEngine',
+      name: 'rightSource',
+    },
+    {
+      class: 'FObjectArray',
+      of: 'org.chromium.webidl.DiffChunk',
+      name: 'diffChunks',
+    },
```

Comment:
These are just added for now. `PipelineMessage` will be broken down into several different classes later this week.